src/singleinstance.py

```diff
@@ -75,7 +75,7 @@ def lock(self):
                     fcntl.lockf(self.fp, fcntl.LOCK_EX | fcntl.LOCK_NB)
                 self.lockPid = os.getpid()
             except IOError:
-                print ('Another instance of this application is already running')
+                print('Another instance of this application is already running')
```

Comment:
`sys.exit('Another instance of this application is already running')`

Reply:
And the same above. `print(e.errno)` is also redundant as for me.

Reply:
Updated